profiles/Pericles001.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/55035365/176567832-8f8412c9-1f05-49cd-bcef-1cde44d0207e.png",
```

Comment:
I recommend more of an image like this. What do you think? :)

![image](https://user-images.githubusercontent.com/51391473/176586265-9f8acc6f-0954-46eb-aa62-ee3a8be7c10a.png)

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/176586265-9f8acc6f-0954-46eb-aa62-ee3a8be7c10a.png",
```



__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question